src/main/java/org/carlmontrobotics/lib199/MotorErrors.java

```diff
@@ -65,47 +70,56 @@ private static <T extends Enum<T>> void reportError(String vendor, T error, T ok
         System.err.println(Arrays.toString(stack));
     }
 
-    public static void checkSparkErrors(CANSparkBase spark) {
+    public static void checkSparkErrors(SparkBase spark) {
         //Purposely obviously impersonal to differentiate from actual computer generated errors
-        short faults = spark.getFaults();
-        short stickyFaults = spark.getStickyFaults();
-        short prevFaults = flags.containsKey(spark) ? flags.get(spark) : 0;
-        short prevStickyFaults = stickyFlags.containsKey(spark) ? stickyFlags.get(spark) : 0;
-
-        if (spark.getFaults() != 0 && prevFaults != faults) {
-        System.err.println("Whoops, big oopsie : fault error(s) with spark id : " + spark.getDeviceId() + ": [ " + formatFaults(spark) + "], ooF!");
+        // short faults = spark.getFaults();
+        Faults faults = spark.getFaults();
+        Faults stickyFaults = spark.getStickyFaults();
+        Faults prevFaults = flags.containsKey(spark) ? flags.get(spark) : null;
+        Faults prevStickyFaults = stickyFlags.containsKey(spark) ? stickyFlags.get(spark) : null;
```

Comment:
Ik this was here from before, but it probably makes sense to do
```suggestion
        Faults prevFaults = flags.getOrDefault(spark, null);
        Faults prevStickyFaults = stickyFlags.getOrDefault(spark, null);